src/iris/ui/static/js/iris.js

```diff
@@ -2026,7 +2033,7 @@ iris = {
         app.editable = window.appData.user_admin || owner;
         app.supportViewingKey = window.appData.user_admin || owner;
         app.supportEditingEmailIncidents = window.appData.user_admin || owner;
-        app.showEditOwners = window.appData.user_admin;
+        app.showEditOwners = window.appData.user_admin || owner;
```

Comment:
Sorry for the variables being named oddly. It would be more appropriate if `owner` was instead named `is_owner` and `user_admin` was instead named `is_admin`.

Its intended for these to be boolean values as they're used in the various if statements in the template to allow certain actions based on whether a user is an owner of this app, or an admin of iris itself.